### PR TITLE
feat: sync-with-qdrant template-python

### LINKED ISSUE #287: 🚀 Enhancement: Template to demonstrate usage with Qdrant
### 🔖 Enhancement description

A template demonstrating usage with Qdrant - https://qdrant.tech/ in Appwrite functions.

### 🎤 Pitch

Greetings from Qdrant 👋

I am opening this issue to propose a template that as the title goes, demonstrates the usage of Appwrite with [Qdrant](https://qdrant.tech).

I have associated a pull requests with this issue.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by Anush008**
## Description
Adds a template to demonstrate the usage of [Qdrant](https://qdrant.tech/) in Appwrite functions. Uses the Python runtime.

~This template is yet to be deployed to an AppWrite function since my builds are stuck in the `waiting` state.~ RESOLVED

## Related PRs and Issues
Resolves #287 along with the other PRs.

### CODE REVIEW

**Comment by loks0n** [comment]
> This template is yet to be deployed to an AppWrite function since my builds are stuck in the `waiting` state.

Are you able to share more details about this?

cc @Meldiron

File: python/sync_with_qdrant/src/utils.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        file_name (str): Name of the file to read
+
+    Returns:
+        (str): Contents of static/{file_name}
+    """
+    file_path = os.path.join(static_folder, file_name)
+    with open(file_path, "r") as file:
+        return file.read()
+
+def get_all_documents():
```
Comment (loks0n) [inline python/sync_with_qdrant/src/utils.py:39]: Can we put this function inside a `appwrite.py` instead? It doesn't feel related to these other functions

**Review by loks0n — CHANGES_REQUESTED** [review]

File: python/sync_with_qdrant/src/utils.py (hunk 1 above)
Comment (Anush008) [reply python/sync_with_qdrant/src/utils.py]: [`25a6fc1` (#289)](https://github.com/appwrite/templates/pull/289/commits/25a6fc1a0f800ffcdef0d61784e3b94719239661)

**Comment by Anush008** [comment]
> > This template is yet to be deployed to an AppWrite function since my builds are stuck in the `waiting` state.
> 
> Are you able to share more details about this?
> 

I retried with a new function deployment. The template works as expected.

**Pull request merged**